### ISSUE TITLE
t/10_new.t started to fail

### ISSUE THREAD

**Issue by eserte** [ytnobody/p5-Net-Azure-EventHubs#2]
I see a new test failure on my smokers:
```
    #   Failed test 'throws "connection_string is required"'
    #   at t/10_new.t line 10.
    # expecting: Regexp ((?^:\Aconnection_string is required))
    # found: endpoint was not specified. Please specify endpoint or connection_string. at /home/cpansand/.cpan/build/2018060721/Net-Azure-EventHubs-0.04-0/blib/lib/Net/Azure/EventHubs.pm line 40.
    #  at /home/cpansand/.cpan/build/2018060721/Net-Azure-EventHubs-0.04-0/blib/lib/Net/Azure/EventHubs.pm line 42.
    # Looks like you failed 1 test of 2.

#   Failed test 'void params'
#   at t/10_new.t line 12.
# Looks like you failed 1 test of 5.
t/10_new.t .................. 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/5 subtests 
```

It seems that this problem started a few days ago:
http://matrix.cpantesters.org/?dist=Net-Azure-EventHubs%200.04;reports=1#sl=7,1

**Comment by ytnobody** [comment]
Thank you for reporting.

It's an affect of changing behavior in Net-Azure-Authorization-SAS-0.03.
I forgot to fix Net-Azure-Eventhub and Net-Azure-NotificationHubs.

I'm going to fix this issue today.

Best regards.

**Comment by ytnobody** [comment]
I fixed this issue and released as version 0.05.

Thank you.
